code/_helpers/matrices.dm

```diff
@@ -3,3 +3,119 @@
 	Turn(.) //BYOND handles cases such as -270, 360, 540 etc. DOES NOT HANDLE 180 TURNS WELL, THEY TWEEN AND LOOK LIKE SHIT
 
 
+/atom/proc/SpinAnimation(speed = 10, loops = -1)
```

Comment:
We already have spinanimation in animations.dm

Reply:
no you dont, i moved it